docs/reference/buildx_build.md

```diff
@@ -359,6 +359,43 @@ $ docker buildx create --use --name insecure-builder --buildkitd-flags '--allow-
 $ docker buildx build --allow security.insecure .
 ```
 
+### <a name="secret"></a> Secret to expose to the build (`--secret`)
+
+```
+--secret=[id=ID[,KEY=VALUE]
```

Comment:
Technically the keys are id, src/source, env and type. If type is unset then it is detected. If source/env is unset then id itself is used. The examples are ok but I think we should also explain the format fully.